### ISSUE TITLE
Add extension download page

### ISSUE THREAD

**Issue by benkutil** [getroadmap/roadmap-site#12]

**Comment by bkingbking** [comment]
Crossrider provides a universal download link and page that auto-detects the browser type and provides the corresponding extension file: 
* Link: http://crossrider.com/download/57890
* Page: http://crossrider.com/install/57890-roadmap.  Crossrider allows limited style / customization.  Anton M. has the login credentials, as well as the details about what is customizable and what isn't.

**Comment by benkutil** [comment]
@bkingbking, will this always be the most up-to-date version of the extension? If so, I’ll use this link.

I’ve been working on a landing page for the extension, but haven't had a chance to implement it yet. Will use this as part of it.

**Comment by bkingbking** [comment]
@benkutil Correct, the URL or page will always be the most up-to-date version.  Here is the tech documentation for [submitting the extension to Safari Extensions Gallery](https://ppmroadmap.atlassian.net/wiki/display/RTD/Safari+Extension+Gallery+Submission) (maintained in Confluence).  Scroll down to "Download URL" row.

**Comment by benkutil** [comment]
![1password browser extensions 20150217](https://cloud.githubusercontent.com/assets/228373/6241668/2fe95b4a-b6ef-11e4-821f-6a32e16ea6cf.png)

**Comment by bkingbking** [comment]
If the extension must be downloaded from Chrome or Safari stores, will RM site auto-detect browser and redirect to the respective store or do we rely on Crossrider universal download?  Here are the [download & instructions on UV](https://ppmroadmap.uservoice.com/knowledgebase/articles/451111-download-and-configure-chrome-firefox-safari-ex).
